languagetool-core/src/main/resources/org/languagetool/resource/spelling_global.txt

```diff
@@ -10061,4 +10061,1404 @@ Ben Ali
+Gottfried August Bürger
+Gustav Landauer
+Hans Magnus Enzensberger
+Heiner Müller
+Heinrich Cornelius Agrippa
+Heinrich Mann
+Helmut Krausser
+Hermann Sudermann
+Hrotsvitha
+Hugo Ball
+Jakob Böhme
+Jakob Wassermann
+Johann Gottfried Herder
+Johann Gottlieb Fichte
+Karl Rahner
+Kurt Schwitters
+Ludwig Meidner
+Manfred Eigen
+Matthias Deutschmann
+Max Klinger
+Moses Mendelssohn
+Nicholas of Cusa
+Ossi Weiner
+Oswald Spengler
+Otto of Freising
+Philipp Melanchthon
+Philipp Otto Runge
+Ricarda Huch
+Robert Hübner
+Rudolf Erich Raspe
+Siegfried Lenz
+Silvo Lahtela
+Stefan George
+Vladimir Budde
+Walter Jens
+Wilhelm Wundt
+Luděk Pachman
+Christa Wolf
+Victor Klemperer
+Wolf Biermann
+Walther Rathenau
+Erich Kästner
+Martin Walser
+Martin Opitz
+Herta Müller
+Wilhelm Ostwald
+Kurt Tucholsky
+Else Lasker-Schüler
+Kwame Nkrumah
+Johan Ludvig Runeberg
+Zachris Topelius
+Georg Büchner
+Alessandro Allori
+Carlo Collodi
+Edward Gibbon
+Edward Young
+Henry Fielding
+Horace Walpole
+James Macpherson
+John Gay
+John Wesley
+Joshua Reynolds
+Laurence Sterne
+Mary Wollstonecraft
+Matthew Gregory Lewis
+Philip Stanhope
+Samuel Johnson
+Thomas Gray
+Tobias Smollett
+William Congreve
+William Cowper
+Daniel Defoe
+Henry St. John
+Mary Andrea Garcia
+Samuel Richardson
+Ann Radcliffe
+Charles Lock Eastlake
+David Ricardo
+Frederick Marryat
+George Canning
+Henry Brougham
+Jane Austen
+John Bowring
+Leigh Hunt
+Percy Bysshe Shelley
+Robert Southey
+Samuel Taylor Coleridge
+Thomas Babington Macaulay
+Thomas Carlyle
+Thomas Robert Malthus
+Thomas Malthus
+William Beckford
+William Godwin
+William Wordsworth
+Giorgos Seferis
+Nikos Kazantzakis
+Odysseas Elytis
+Vasilios Kotronias
+Yannis Ritsos
+Eleftherios Venizelos
+Yelena Dembo
+Miguel Ángel Asturias Rosales
+Andreas Vesalius
+Gerardus Mercator
+John Amos Comenius
+Felix Dahn
+Athanasius Kircher
+Hans Jakob Christoffel von Grimmelshausen
+Johann Reuchlin
+Ulrich von Hutten
+Willibald Pirckheimer
+Klemens von Metternich
+Jan Hus
+Béla Balázs
+Endre Ady
+Ferenc Molnár
+Mór Jókai
+Brassaï
+Anna Seghers
+György Kepes
+Susan Polgar
+Viswanathan Anand
+Aurobindo Ghosh
+Bhimrao Ramji Ambedkar
+Indira Gandhi
+Jawaharlal Nehru
+Gershom Scholem
+Martin Buber
+Shmuel Yosef Agnon
+Adolivio Capece
+Alberto Moravia
+Alessandro Baricco
+Andrea Camilleri
+Antonio Negri
+Claudio Magris
+Claudio Pantaleoni
+Federico Fellini
+Francesco Clemente
+Franco Battiato
+Giorgio Agamben
+Reinhold Messner
+Salvator Rosa
+Giuseppe Ungaretti
+Giorgio de Chirico
+Gianni Rodari
+Alberto Savinio
+Benedetto Croce
+Carlo Carrà
+Carlo Ginzburg
+Carlo Levi
+Carlo Maria Martini
+Cesare Pavese
+Curzio Malaparte
+Dino Buzzati
+Elsa Morante
+Ermanno Olmi
+Gianni Vattimo
+Ignazio Silone
+Julius Evola
+Leonardo Sciascia
+Luchino Visconti
+Maria Montessori
+Mario Soldati
+Michelangelo Antonioni
+Natalia Ginzburg
+Norberto Bobbio
+Salvatore Quasimodo
+Umberto Eco
+Antonio Tabucchi
+Akira Kurosawa
+Akira Toriyama
+Haruki Murakami
+Kenji Miyazawa
+Kenzaburō Ōe
```

Comment:
Japanese Romanisation conventions are very inconsistently observed. I'd add the forms without the long vowel marks alongside those (so `Kenzaburo Oe`), and potentially also the kana-influenced long vowel digraphs, e.g. `Kenzaburou Ooe`.